service/controller/key/key.go

```diff
@@ -107,7 +164,15 @@ func IsSameApp(component releasev1alpha1.ReleaseSpecComponent, app applicationv1
 		GetComponentRef(component) == app.Spec.Version
 }
 
-func ComponentCreated(component releasev1alpha1.ReleaseSpecComponent, apps []applicationv1alpha1.App) bool {
+func IsSameConfig(component releasev1alpha1.ReleaseSpecComponent, config corev1alpha1.Config) bool {
+	_, managedByReleaseOperator := config.Labels[LabelManagedBy]
```

Comment:
This looks like it's only checking label existence but not the value. What's the point?